getting-started-expressjs/package.json

```
@@ -0,0 +1,22 @@
+{
+  "type": "module",
+  "name": "getting-started-express-js",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js"
+  },
+  "author": "Ahmed Ibrahim",
+  "license": "ISC",
+  "dependencies": {
+    "@metaplex-foundation/js-next": "^0.10.0",
```

Sorry forgot to tell you but, recently, `js-next` has been renamed to `js` since it is now the current SDK for Metaplex.

```suggestion
    "@metaplex-foundation/js": "^0.11.5",
```

A good task for another PR actually would be to update the other examples if you're up for it? 🙂

---

Yes, Sure. Will work on it.